### PR TITLE
COMMERCE-9583 Expose ChannelID lookup in Commerce delivery API

### PR DESCRIPTION
**Pull request by ghost**
https://issues.liferay.com/browse/COMMERCE-9583

### CODE REVIEW

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-impl/rest-openapi.yaml [hunk 1]
```diff
@@ -629,6 +650,87 @@ info:
...
+                            schema:
+                                $ref: "#/components/schemas/Error"
+                    description:
+                        Unexpected error
+            security:
+                - BasicAuth: []
+                - BearerAuth: []
+                - OAuth2:
+                      - CommerceOpenApiAdmin.read
+            summary: "Gets a list of channel."
```
Comment (luca-pellizzon) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-impl/rest-openapi.yaml:731]: should be "channels" plural

**Review by luca-pellizzon — CHANGES_REQUESTED** [review]
One description to fix and commits to sort better.
Regarding permissions, we are so relying on VIEW permission assigned at the model level right?
cc @andrea-ale-sbarra

**Comment by andrea-ale-sbarra** [comment]
> One description to fix and commits to sort better. Regarding permissions, we are so relying on VIEW permission assigned at the model level right? cc @andrea-ale-sbarra

@luca-pellizzon we should, that's what we decided during the grooming

**Comment by andrea-ale-sbarra** [comment]
resend here https://github.com/liferay-commerce/liferay-portal/pull/2920

**Pull request closed without merging**